collector/collector.go

```diff
@@ -51,13 +53,21 @@ func RecordMetrics(username string, password string, netappH string, netappP int
 	}()
 }
 
-func connectToHost(username string, password string) (*ssh.Client, error) {
+func connectToHost(username string, password string, knownHostsFile string) (*ssh.Client, error) {
 	sshConfig := &ssh.ClientConfig{
 		User:    username,
 		Auth:    []ssh.AuthMethod{ssh.Password(password)},
 		Timeout: time.Second * 30,
 	}
 	sshConfig.HostKeyCallback = ssh.InsecureIgnoreHostKey()
+	if knownHostsFile != "" {
+		callback, err := knownhosts.New(knownHostsFile)
+		if err != nil {
+			fmt.Println(err)
```

Comment:
You should use a logger instead of fmt :slightly_smiling_face:

Reply:
Done ✅